pkg/cmdimpl/add.go

```diff
@@ -235,6 +237,26 @@ func generateSourceManifest() ([]byte, error) {
 	return sourceManifest, nil
 }
 
+func generateSourceManifestHelm() ([]byte, error) {
+	if params.Chart == "" {
+		return nil, fmt.Errorf("chart needs to be set")
+	}
```

Comment:
I added this check in the source because i figured we didnt want to make the source only to have the manifest fail. Pretty sure there might be a better way to check if the chart is valid but I figured i would just get it done for now.